CIScripts/Test/Tests/Protocol/SingleHostWithUtils.Tests.ps1

```diff
@@ -130,15 +137,26 @@ Describe "Single compute node protocol tests with utils" {
     }
 
     AfterEach {
-        Write-Host "Removing containers"
-        if (Get-Variable Container1ID -ErrorAction SilentlyContinue) {
-            Invoke-Command -Session $Session -ScriptBlock { docker rm -f $Using:Container1ID } | Out-Null
-            Invoke-Command -Session $Session -ScriptBlock { docker rm -f $Using:Container2ID } | Out-Null
+        Write-Host "Removing container 1"
+        if (Get-Variable "Container1ID" -ErrorAction SilentlyContinue) {
+            Invoke-NativeCommand -Session $Session -CaptureOutput {
```

Comment:
I wonder if we still need ```| Out-Null``` here, since when I try to do something like this:
```Invoke-NativeCommand -CaptureOutput {whoami.exe}``` I get some output in my console.